docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -1032,21 +1032,21 @@ Moreover, don't forget to add Google as a *Source* from your connection profile
 
 Facebook
 ^^^^^^^^
-To use Facebook as an authentication source, you also need an API code and a secret key.  To get one, go here: https://developers.facebook.com/apps.  When you create your App, make sure you specify the following as the Website URL:
-https://YOUR_PORTAL_HOSTNAME/oauth2/callback
-Of course, replace the hostname with the values from `general.hostname` and `general.domain`.
+To use Facebook as an authentication source you will need to setup a Facebook app. Go to: <https://developers.facebook.com/apps>.
+
+Click the *Add a New APP* button or click on an existing APP. Choose a _Display Name_ and a _Contact Email_ address, then click on *Create APP ID*. On the following page in *Add a Product* for *Facebook Login* by clicking *Set Up*. On the following page select *WWW* (Web) as the _Platform_. Enter the *Site URL*: <https://[`HOSTNAME`].[`DOMAIN`]/>. Click *Save*, then click *Continue*.
 
-To find the secret, go in your newly created app, and click on *Settings -> Basic*.
+NOTE: Go to *Configuration -> System Config -> General Configurtation* for the `HOSTNAME` and `DOMAIN` to use throughout the Facebook OAuth configuration (example: *HOSTNAME*=`packetfence.moc` *DOMAIN*=`pf810`, therefore *Site URL*=`https://pf810.packetfence.moc/`).
 
-While in *Settings -> Basic*, add YOUR_PORTAL_HOSTNAME in the *App Domains* field. Next, you will need to add the product *Facebook Login*. Click on *Set up*, and choose *Web* platform. Go through the 5 steps, then on the left side of the screen, go in settings under Facebook Login. For *Valid OAuth Redirect URIs*, enter https://YOUR_PORTAL_HOSTNAME/oauth2/callback and then save changes.
+Navigate to *Product -> Facebook Login -> Settings*. Enable *Web OAuth Login* and under *Valid OAuth Redirect URIs* enter the following URI using your `HOSTNAME` and `DOMAIN`: <https://[`HOSTNAME`].[`DOMAIN`]/oauth2/callback>
 
-Also, add the following Authorized domains : *.facebook.com, *.fbcdn.net, *.akamaihd.net (May change)
+Navigate to *Main Menu -> Settings -> Basic*. Note the *App ID* and *App Secret*. In *App Domains* enter the following URI: [`HOSTNAME`].[`DOMAIN`] (example: *HOSTNAME*=`packetfence.moc` *DOMAIN*=`pf810`, therefore *App Domains*=`pf810.packetfence.moc`).
+
```

Comment:
(example: *HOSTNAME*=`pf810` *DOMAIN*=`packetfence.moc`,

Reply:
Fixed in https://github.com/inverse-inc/packetfence/pull/3751/commits/b1d5de43119847bdf20211e34a7b5b860ae9e2e1